src/Components/Members/index.js

```diff
@@ -1,25 +1,126 @@
 import { useEffect, useState } from 'react';
 import styles from './members.module.css';
+//components imports
+import Table from './Table/index';
+import AddForm from './AddForm/index';
+import UpdateForm from './UpdateForm/index';
 
 function Members() {
+  const [showAddMember, setShowAddMember] = useState(false);
+  const [showUpdMember, setShowUpdMember] = useState(false);
   const [members, setMembers] = useState([]);
+  const [selectId, setSelectId] = useState('');
+
+  const getMembs = async () => {
+    const response = await fetch(`${process.env.REACT_APP_API_URL}/api/member`);
+    const data = await response.json();
+    data.data.forEach((d) => {
+      d.birthDate = d.birthDate.substring(0, 10);
+    });
+    setMembers(data.data);
+  };
 
   useEffect(() => {
-    fetch(`https://jsonplaceholder.typicode.com/users`)
-      .then((response) => response.json())
-      .then((response) => {
-        setMembers(response);
-      });
+    getMembs();
   }, []);
 
+  const addMember = async (member) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/member`, {
+        method: 'POST',
+        headers: { 'Content-type': 'application/json' },
+        body: JSON.stringify(member)
+      });
+
+      const newMemb = await response.json();
```

Comment:
We are not cheking here for the respone error, so if i send an invalid DNI it will tell me the success message but the request will fail and the member is not save on the DB

Reply:
Solved